bootstrap/kubeadm/types/upstreamv1beta3/conversion_test.go

```diff
@@ -87,6 +88,15 @@ func joinConfigurationFuzzer(obj *JoinConfiguration, c fuzz.Continue) {
 	obj.SkipPhases = nil
 }
 
+func bootstrapv1JoinConfigurationFuzzer(obj *bootstrapv1.JoinConfiguration, c fuzz.Continue) {
+	c.FuzzNoCustom(obj)
+
+	// JoinConfiguration.Discovery.File.KubeConfig is internal to Cluster API and does not exist in kubeadm v1beta1 types, pinning it to avoid round trip errors.
```

Comment:
Nit, we recently removed all the comments similar to this one and added one for all at L74-76 (you can drop the comment); same applies to conversion_tests.go files for other versions of the kubeadm types